Sources/Datadog/DDRUMMonitor.swift

```diff
@@ -109,6 +111,19 @@ public class DDRUMMonitor {
         attributes: [AttributeKey: AttributeValue] = [:]
     ) {}
 
+    /// Notifies that the Resource starts being loaded using GET request to given `url`.
+    /// - Parameters:
+    ///   - resourceKey: the key representing the Resource - must be unique among all Resources being currently loaded.
+    ///   - httpMethod: HTTP method used to fetch resource
+    ///   - urlString: the `URL` for the Resource in `String` form.
+    ///   - attributes: custom attributes to attach to the Resource.
+    public func startResourceLoading(
+        resourceKey: String,
+        httpMethod: String,
```

Comment:
In #366 we merged public `RUMMethod` enumeration, which should be used here.

Reply:
`internal enum RUMHTTPMethod` should be removed then
i don't remember why it was created in the first place 🤔

Reply:
It was there because the old version of rum data models used two different representations of `method` (one for `resource.method` and another one for `error.resource.method`. I think we can remove it as part of the JIRA I suggest in https://github.com/DataDog/dd-sdk-ios/pull/369#discussion_r557228073. This requires update in `rum-models-generator` to rename it from `RUMMethod` to `RUMHTTPMethod` (which sounds better).

Reply:
i just removed it and replaced it with `RUMMethod` ✅ it was trivial

i think `RUMMethod` is okay because properties too have the name `method` (`Resource.method: RUMMethod`)
if we rename it, IMO we should also rename properties as well